### PR TITLE
fruits are written into fruits.csv

### PR DESCRIPTION
**Pull request by ZsoltToth**
I wrote my favorite fruits into this file

### CODE REVIEW

File: network/src/resources/fruits.csv [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Apple
+Pear
+Grape
+Cucumber
```
Comment (judit0310) [inline network/src/resources/fruits.csv]: It is not a fruit

File: network/src/resources/fruits.csv [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+Potato
```
Comment (judit0310) [inline network/src/resources/fruits.csv]: It is not a fruit

**Review by judit0310 — CHANGES_REQUESTED** [review]
Correct fruits.csv for containing only fruits

File: network/src/resources/fruits.csv (hunk 1 above)
Comment (ZsoltToth) [reply network/src/resources/fruits.csv]: OK, U R Right

File: network/src/resources/fruits.csv (hunk 2 above)
Comment (ZsoltToth) [reply network/src/resources/fruits.csv]: Really??? Potato is life. :)

Comment (judit0310) [reply network/src/resources/fruits.csv]: But not fruit

Comment (marwane323) [reply network/src/resources/fruits.csv]: > Really??? Potato is life. :)

But still it's not a fruit
:v

**Review by judit0310 — APPROVED** [review]
It's ok now.

**Pull request merged**